nautobot/docs/user-guide/administration/installation/index.md

```diff
@@ -1,24 +1,95 @@
 # Installation
 
-Nautobot can be deployed via Docker (Compose or Kubernetes) or directly onto a supported Linux system:
+Whether you are getting started for the first time, looking to build a local lab or ready to build a production environment, there are a variety of options to find the right fit for your needs with using Nautobot. 
 
-* [Nautobot Docker images](../guides/docker.md) are available via [Docker Hub](https://hub.docker.com/r/networktocode/nautobot) & [GitHub Container Registry](https://github.com/nautobot/nautobot/pkgs/container/nautobot) for use within a containerized environment
-    * Install Nautobot via [Docker Compose](https://github.com/nautobot/nautobot-docker-compose)
-    * [Install Nautobot via Helm Charts](https://docs.nautobot.com/projects/helm-charts/en/stable/) for Kubernetes
-* [Install Nautobot](install_system.md) onto a [RHEL (Red Hat Enterprise Linux)](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux) or [Ubuntu](https://ubuntu.com/) virtual machine.
+!!! tip "Checking out Nautobot for the first time? Not ready to install anything yet?"
+    You can explore the [Nautobot Always-On Sandbox Environments](https://networktocode.com/nautobot/sandbox-environments/). There is sample data already prepopulated, no installation required, and several Nautobot Apps are pre-installed, with sample data.
 
-For more information about the Docker tags, Docker configurations, or using container images for your own development environment, see [Nautobot in Docker](../guides/docker.md).
+## Deployment Options
 
-??? info "Supported Platforms"
+=== "Bare metal / Virtual Machine"
 
+    You can [install Nautobot](install_system.md) onto a [RHEL (Red Hat Enterprise Linux)](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux) or [Ubuntu](https://ubuntu.com/) virtual machine. 
+    
     Nautobot was designed to be a cross-platform application that can run on nearly any system that is able to run the required dependencies. *Only the operating system platforms listed below are officially supported at this time*.
 
-    - Red Hat flavors of Linux including CentOS 8.2+ and Red Hat Enterprise Linux (RHEL) 8.2+ are supported.
-    - Debian/Ubuntu flavors of Linux including Ubuntu 20.04+ and Debian 11+ are supported.
+    * Red Hat flavors of Linux including CentOS 8.2+ and Red Hat Enterprise Linux (RHEL) 8.2+ are supported.
+    * Debian/Ubuntu flavors of Linux including Ubuntu 20.04+ and Debian 11+ are supported.
```

Comment:
we should update this from 20.04+ to at least 22.04 and evaluate all of these.

Reply:
We haven't dropped support for 20.04 as far as I know.

Reply:
Maybe we change the phrasing from `support` to recommended?

Reply:
should we keep this as is, but remove the Centos per Josh's recommendation?

Reply:
@glennmatthews @itdependsnetworks Any feedback regarding these statements or should I resolve this comment?
Thanks!